prismic-model/src/exhibition-guides.ts

```diff
@@ -27,7 +27,7 @@ const exhibitionGuides: CustomType = {
       components: list('Guide Component', {
         number: number('Position number'),
         title,
-        creator: structuredText('Creator', 'single'),
+        tombstone: structuredText('Tombstone', 'single'),
```

Comment:
Maybe link to the Slack thread here? Because "tombstone" is a bit of an unusual name imo.